src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -790,87 +790,89 @@ export function useSwapInputsController({
       const didInputAssetChange = current.assetToSellId !== previous?.assetToSellId;
       const didOutputAssetChange = current.assetToBuyId !== previous?.assetToBuyId;
 
-      if (didInputAssetChange || didOutputAssetChange) {
-        const balance = internalSelectedInputAsset.value?.maxSwappableAmount;
-
-        const areBothAssetsSet = internalSelectedInputAsset.value && internalSelectedOutputAsset.value;
-        const didFlipAssets =
-          didInputAssetChange && didOutputAssetChange && areBothAssetsSet && previous && current.assetToSellId === previous.assetToBuyId;
-
-        if (!didFlipAssets) {
-          // If either asset was changed but the assets were not flipped
-          if (!balance || equalWorklet(balance, 0)) {
-            isQuoteStale.value = 0;
-            isFetching.value = false;
-            inputValues.modify(values => {
-              return {
-                ...values,
-                inputAmount: 0,
-                inputNativeValue: 0,
-                outputAmount: 0,
-                outputNativeValue: 0,
-              };
-            });
-            return;
-          }
+      if (!didInputAssetChange && !didOutputAssetChange) return;
```

Comment:
this looks like a lot of changes but it is mostly formatting as I'm just returning early here instead of keeping everything inside of an if statement